docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -13,31 +13,26 @@
 
 If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+1. Check your operating system and its version: 
```

Comment:
I do not think this step is necessary and if it is - it must be part of GPU driver manual probably.
Ideally we should just fall back to Gpu driver page and that's it.